src/documentation/pages/components/copy-text/index.vue

```diff
@@ -0,0 +1,32 @@
+<template lang="pug">
+  <div>
```

Comment:
Maybe this div is not needed?

Reply:
Vue3 does not require to have only one parent element in the templates any longer. However, for pages it's needed in for page transitions. This div I would keep just in case, since this is the page for the component.